.neuro/live.yml

```diff
@@ -0,0 +1,50 @@
+kind: live
+title: Fooocus
+
+defaults:
+  life_span: 5d
+  preset: a100x1
+
+images:
+  fooocus:
+    ref: ghcr.io/neuro-inc/fooocus:latest
+  fooocus_apolo:
+    ref: image:${{ flow.project_id }}
+    context: ${{ flow.workspace }}
+    dockerfile: ${{ flow.workspace }}/Dockerfile
+    build_preset: cpu-medium
+volumes:
+  models:
+    remote: storage:/$[[ project.project_name ]]/$[[ flow.project_id ]]/data/models
+    mount: /content/data/model
+    local: models
+  outputs:
+    remote: storage:/$[[ project.project_name ]]/$[[ flow.project_id ]]/app/outputs
+    mount: /content/app/outputs
+
+jobs:
+  fooocus:
+    # image: ${{images.fooocus_apolo.ref}}
+    image: ${{images.fooocus.ref}}
+    name: fooocus
+    preset: a100x1
+    http_port: 7865
+    browse: true
+    volumes:
+      - ${{volumes.models.ref_rw}}
+      - ${{volumes.outputs.ref_rw}}
+    env:
+      CMDARGS: --listen # Arguments for launch.py.
+      DATADIR: /content/data # Directory which stores models, outputs dir
```

Comment:
should it be mounted from storage dir?

Reply:
solved this by replacing the `models` volume with a `data` volume that I can pass here with `${{ volumes.data.mount }}` as you suggested below